diff --git a/dist/electron-version.js b/dist/electron-version.js
index 55800845e873a05b04603f55b8bcd0639a8489a6..09d92d2a14c6b480be731a18de558a7f344cb1df 100644
--- a/dist/electron-version.js
+++ b/dist/electron-version.js
@@ -28,11 +28,8 @@ async function findAncestorNodeModulesPath(dir, packageName) {
     return Promise.resolve(undefined);
 }
 async function determineNodeModulesPath(dir, packageName) {
-    const nodeModulesPath = path_1.default.join(dir, 'node_modules', packageName);
-    if (await fs_extra_1.default.pathExists(nodeModulesPath)) {
-        return nodeModulesPath;
-    }
-    return findAncestorNodeModulesPath(dir, packageName);
+    const nodeModulesPath = require.resolve(packageName, { paths: [dir] });
+    return path_1.default.dirname(nodeModulesPath);
 }
 class PackageNotFoundError extends Error {
     constructor(packageName, dir) {
