diff --git a/dist/useProgramFromProjectService.js b/dist/useProgramFromProjectService.js
index 7c4078d8480c6020fae2ce44e66613d81dc0f842..ca3e852edb426e1714fa21e77093be1f7729b615 100644
--- a/dist/useProgramFromProjectService.js
+++ b/dist/useProgramFromProjectService.js
@@ -43,6 +43,7 @@ function useProgramFromProjectService({ allowDefaultProjectForFiles, maximumDefa
         log('Could not find project service program for: %s', filePathAbsolute);
         return undefined;
     }
+    if (!opened.configFileName)
     defaultProjectMatchedFiles.add(filePathAbsolute);
     if (defaultProjectMatchedFiles.size > maximumDefaultProjectFileMatchCount) {
         throw new Error(`Too many files (>${maximumDefaultProjectFileMatchCount}) have matched the default project.${validateDefaultProjectForFilesGlob_1.DEFAULT_PROJECT_FILES_ERROR_EXPLANATION}
